README.md

```diff
@@ -18,6 +18,11 @@ First time running the project? Make sure you have Python .
 ## Python Setup
 
 * Install [python](https://www.python.org/)
+* Python version is follow Heroku supported version(now python-3.6.6),if you want specify a Python version need add a runtime.txt
+   ```
```

Comment:
I think we should just do something like this?

Python 3.6.6 or greater? or just Python 3.6.6

Reply:
oh~
this PR hava runtime.txt so ,Python version is follow runtime.text

Reply:
And what has so many commits 
because I update
1. parser.html only hava parser function
2. add setting html hava setting function
3. add Chrome driver, Let we can load url to complete website

Reply:
Okay, I see. I'm actually Okay with these. If @mathsigit doesn't have any comments, this PR can be merged then. Thank you @wu87988622 :)